char_counter.cpp

```diff
@@ -0,0 +1,44 @@
+#include<conio.h>
+#include<iostream>
+#include<string.h>
+#include<ctype.h>
+#include<stdio.h>
+using namespace std;
+
+//progrram to count the number of alphabets, vowels, digits and words in a string
+
+  int main()
+   {
+    system("cls");
+    int i,alphacount=0,vowelcount=0,degitcount=0,wordcount=0;
+    char a[50];  //storing the string
+    cout<<"\tEnter a string to count\n\t";
+    gets(a);
```

Comment:
[Why is the gets function so dangerous that it should not be used? - StackOveflow](https://stackoverflow.com/questions/1694036/why-is-the-gets-function-so-dangerous-that-it-should-not-be-used)